service/dap/server.go

```diff
@@ -1529,12 +1525,20 @@ func (s *Server) toClientPath(path string) string {
 	if len(s.args.substitutePathServerToClient) == 0 {
 		return path
 	}
-	return locspec.SubstitutePath(path, s.args.substitutePathServerToClient)
+	clientPath := locspec.SubstitutePath(path, s.args.substitutePathServerToClient)
+	if clientPath != path {
+		s.log.Debugf("server path=%s converted to client path=%s in stack trace request\n", path, clientPath)
```

Comment:
Please remove "in stacktrace request" since this is a general logging message and can be used in other contexts